src/main/java/model/CalculatorType.java

```diff
@@ -0,0 +1,31 @@
+package model;
+
+import java.util.Arrays;
+import java.util.Optional;
+import java.util.function.BiFunction;
+
+public enum CalculatorType {
+    ADD("+",(left,right) -> left + right),
+    SUBTRACT("-",(left,right) -> left - right),
+    MULTIPLY("*", (left,right) -> left * right),
+    // 나중에 분모 0 처리 방법 고민해보기
+    DIVIDE("/", (left,right) -> left / right);
```

Comment:
```suggestion
    // 나중에 분모 0 처리 방법 고민해보기
    DIVIDE("/", (left,right) -> {
       if(분모가 0이냐?) { 에러처리 }
       return left / right
    });
```

이런식으로 사용한다면 0도 처리 가능하겠네요 :)